backend/app/api/endpoints/admin.py

```
@@ -1024,3 +1028,118 @@ async def update_quick_access_config(
+        {
+            "id": 5,
+            "zh": "我可以解释复杂的代码逻辑，帮助你理解代码库",
+            "en": "I can explain complex code logic and help you understand the codebase",
+        },
+    ],
+}
+
+
+@router.get("/system-config/slogan-tips", response_model=ChatSloganTipsResponse)
+async def get_slogan_tips_config(
+    db: Session = Depends(get_db),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)